scripts/gen_reports.py

```diff
@@ -67,12 +67,29 @@ def get_heading_report_values(headings, oss_entity):
 
         diff_color = ''
```

Comment:
I made a truth table to hopefully make the logic easier to understand. LMK if this is too spaghetti please @natalialuzuriaga @decause-gov